docs/jitserver_tuning.md

```diff
@@ -120,6 +120,11 @@ A value greater than 0 for the `Compilation Queue Size` is a sign that the serve
 
     -XcompilationThreads<N> (default is 63)
 
-More detailed diagnostics can be obtained with the option `-Xjit:verbose={JITServer},verbose={compilePerformance}` which is typically used for debugging server behavior.
+![Start of content that applies to Java 8+](cr/java8plus.png) You can specify a larger number of threads with `-XcompilationThreads<N>` on the JITServer (999) and the JIT client (16). The default number of threads is kept at 63. 
```

Comment:
I suggest to compact all this information:

-XcompilationThreads<N> (default is 63, maximum is 999)

Increasing the maximum number of client threads (maximum is 15) can improve performance in high network latency settings since there can be more in-flight concurrent compilation requests. Increasing the number of threads at the server can improve performance if it has many CPU cores available and serves a large number of clients concurrently.